assets/charts/components/prometheus-operator/values.yaml

```diff
@@ -1125,6 +1125,7 @@ prometheus-node-exporter:
   extraArgs:
     - --collector.filesystem.ignored-mount-points=^/(dev|proc|sys|var/lib/docker/.+)($|/)
     - --collector.filesystem.ignored-fs-types=^(autofs|binfmt_misc|cgroup|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|mqueue|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|sysfs|tracefs)$
+    - --no-collector.powersupplyclass
```

Comment:
Do you mind adding a line or two above this to give context on why this was disabled? Just so that we don't lose sight of it. Maybe we can say something like Flatcar specific change or something :wink: .

Reply:
I did add in the PR and the commit message.

I will add in the file as well.

Reply:
done.